SampleGeneration/20250228_ForestReducer_HFJet_2023OldReco/RunParallelData23.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+MAXCORES=100
```

Comment:
Quick thing that I spotted. Lets reduce this <60 (issue spotted by Cristian when running too many parallel jobs)

Reply:
Thanks for the catch! I reduced it to 40, which is what I think @jdlang  set the others to!